compiler-rt/lib/hwasan/hwasan_report.cpp

```diff
@@ -373,7 +375,7 @@ static void ShowHeapOrGlobalCandidate(uptr untagged_addr, tag_t *candidate,
   }
 }
 
-void PrintAddressDescription(
+static void PrintAddressDescription(
```

Comment:
Why don't we pull all the static into the anon namespace

Reply:
There is some LLVM rule to prefer static over {}
but it will be a method anyway

Reply:
For posterity: https://llvm.org/docs/CodingStandards.html#anonymous-namespaces

> Because of this, we have a simple guideline: make anonymous namespaces as small as possible, and only use them for class declarations. For example: